platform_versioned_docs/version-22.4/compute-envs/aws-batch.mdx

```diff
@@ -216,7 +216,7 @@ Jump to the documentation for [launching pipelines](../launch/launchpad.mdx).
 - You can set **Min CPUs** to be greater than `0`, in which case some EC2 instances will remain active. An advantage of this is that pipeline executions will initialize faster.
 
   :::caution
-  Keeping EC2 instances running may result in additional costs. You will be billed for these running EC2 instances regardless of whether you are executing pipelines or not.
+  Setting Min CPUs to a value greater than 0 will keep the required compute instances active, even when not running your pipeline executions. Note that this will result in additional AWS charges.
```

Comment:
```suggestion
  Setting Min CPUs to a value greater than 0 will keep the required compute instances active, even when your pipelines are not running. This will result in additional AWS charges.
```

Reply:
Suggestion.